fsharp-backend/src/Prelude/Prelude.fs

```diff
@@ -904,6 +904,27 @@ module Json =
       override _.Write(writer : Utf8JsonWriter, _ : Password, _options) =
         writer.WriteStringValue("Redacted")
 
+    // Since we're getting this back from OCaml in DDates, we need to use the
+    // timezone even though there isn't one in the type
+    type LocalDateTimeConverter() =
```

Comment:
I just copied and modified the one from OCamlCompat.

Reply:
thought about re-using some code, but decided against it (rule of 3)

Reply:
We'll delete the old one at some point I hope.

Reply:
exactly